docs/deployment_patterns/how_to_use_great_expectations_with_prefect.md

```diff
@@ -82,7 +82,7 @@ Here is an example with an in memory `DataContext`:
 import os
 from pathlib import Path
 
-import great_expectations as ge
+import great_expectations as gx
```

Comment:
probably some additional work required in this file

Reply:
this was a weird one but I manually reviewed it and think we're good? Doesn't look we ever actual use the `gx`/`ge` import?

Reply:
then can we remove it?